packages/strapi/lib/commands/develop.js

```diff
@@ -39,7 +40,7 @@ module.exports = async function({ build, watchAdmin, browser }) {
     if (cluster.isMaster) {
       if (watchAdmin) {
         try {
-          execa('npm', ['run', '-s', 'strapi', 'watch-admin', '--', '--browser', browser], {
+          execa('npm', ['run', '-s', 'strapi', 'watch-admin', '--', '--browser', browser, '--repl', repl], {
```

Comment:
why do you pass the repl param here is it is not supported in the watch-admin command ?